identity-service/src/routes/userBankTransactionMetadata.js

```diff
@@ -0,0 +1,94 @@
+const models = require("../models");
+const {
+  handleResponse,
+  successResponse,
+  errorResponseBadRequest,
+  errorResponseServerError,
+} = require("../apiHelpers");
+const { logger } = require("../logging");
+const { Op } = require("sequelize");
+
+const requiredPurchaseKeys = {
+  discriminator: { type: "string" },
+  usd: { type: "string" },
+  sol: { type: "string" },
+  audio: { type: "string" },
+  swapTransaction: { type: "string" },
+  buyTransaction: { type: "string" },
```

Comment:
discussed on huddle, probably best to just have this as an object to give us more flexibility

Reply:
(assuming you mean swapTransaction here?)

Reply:
I opted to keep the json flat, if we want to add setup/cleanup txs we can do them as top level items